drivers/kscan/kscan_ite_it8xxx2.c

```diff
@@ -19,6 +21,8 @@
 #define LOG_LEVEL CONFIG_KSCAN_LOG_LEVEL
 LOG_MODULE_REGISTER(kscan_ite_it8xxx2);
 
+#define KEYBOARD_KSI_PIN_COUNT		0x8
```

Comment:
This count can be determined by the `IT8XXX2_DT_INST_WUCCTRL_LEN(0)`.

Reply:
Done